### DISCUSSION TITLE
How to load css styles to a specific component and not globally?

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#15577] [Help] [answered]
Hello! Now I have been wondering, Is there any way to load a css stylesheet for a specific component and not globally?
I have tried to: 1) import the css stylesheet into the file, 2) use the `<link>` tag to include the `.css` file. But none of them worked

**Comment by jamesmosier** [comment, marked as answer]
Hi @nspeedzy. This is a heavily requested feature. There is an RFC that should cover something similar to this https://github.com/vercel/next.js/discussions/15208

or a few other resources with information: 

https://github.com/vercel/next.js/discussions/15513
https://github.com/vercel/next.js/issues/13282
https://github.com/vercel/next.js/discussions/14672#discussioncomment-31392
https://github.com/vercel/next.js/issues/12079
https://github.com/vercel/next.js/issues/10975

**Reply by ghost** [reply]
Ok I got you. Thanks for the heads up!
